### PR TITLE
Add basic pages and create react components

### PR DESCRIPTION
**Pull request by edgardsoza**
Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
Create a directory for your reusable components: /components. Done
Install [React Router](https://reactrouter.com/en/main/start/tutorial#setup) Done
[Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App> Done
Add 2 routes in your application: 
/, which contains
A book list, where each book has a title and author Done
A form that allows for creating a new book Done
/categories Done
This should display a button that checks for a status Done
Include a navigation component that includes links to the 2 routes you created Done

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              {item.title}
+              <br />
+              {item.Author}
+              <br />
+              <br />
+            </li>
+          ))}
+        </div>
+        <hr />
+      </div>
```
Comment (AmaduKamara) [inline src/components/BookList.js]: Good job creating a BookList component. According to the project requirements, you are expected to structure your components in the order below:

- [ ] Create a book component that you import in this BookList component, loop through the books array, and pass every single book into the single Book component as a prop, in which you will then receive the prop data inside the Book component (title, author) and render them in that component. Kindly refactor your code this way. This makes the BookList component lean, maintainable, and readable. 👍🏽
- [ ] In this single Book.js component, after rendering the props data(title, and author), there would be a button to delete or remove a book, Kindly add that 👍🏽

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <div className="form-container">
+        <h3 className="form-title">ADD NEW BOOK</h3>
+        <form className="addabook">
+          <input type="text" name="title" placeholder="Book title" />
+          <input type="text" name="author" placeholder="Author" />
+          <input type="submit" value="ADD BOOK" />
+        </form>
+      </div>
+    </div>
```
Comment (AmaduKamara) [inline src/components/BookList.js]: - [ ] For this section, you are expected to have a separate component called AddBook or so, that is responsible for adding a book, then importing this component and rendering it in this file. Kindly move these codes into an AddBook.js component and use that component in space of these codes 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @edgardsoza,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ used create-react-app to initialize the project 👍🏽
✅ configured react-router-dom correctly 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by didierganthier — APPROVED** [review]
Hi @edgardsoza ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Gitflow ✔️ 

## Optional Suggestions 💡
- N/A

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**